solutions/devsprint-cecilia-saraiva-1/GitHubApp/Screens/List/RepositoryCell/RepositoryCellView.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  RepositoryCellView.swift
+//  GitHubApp
+//
+//  Created by Pierre Youness on 26/07/2022.
+//
+
+import UIKit
+
+struct RepositoryCellVM {
+  let name: String
+  let owner:  String
+}
+
+class RepositoryCellView: UITableViewCell {
```

Comment:
[suggestion] if we are not in herding, then it's a good practice to add `final`.
```suggestion
final class RepositoryCellView: UITableViewCell {
```